go.mod

```diff
@@ -49,17 +49,26 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/Masterminds/sprig v2.22.0+incompatible
```

Comment:
why this is a separate line? Can it be moved into the groups?

Reply:
Not sure, just is how `go mod tidy` inserted it.

I put it into the first group and ran the tidy again and now it's good.